kiva/agg/tests/test_affine_matrix.py

```diff
@@ -30,7 +30,8 @@ def test_init_from_array1(self):
         a = ones(6, "D")
         try:
             agg.AffineMatrix(a)
-        except NotImplementedError:
+        except (NotImplementedError, TypeError):
+            # NotImplementedError in Swig 3, TypeError in Swig 4
```

Comment:
I _think_ this is relevant to this?: https://github.com/swig/swig/pull/1261
changelog entry for the above PR:
"2018-07-31: wsfulton
            [Python] #1293 Overloaded C++ function wrappers now raise a TypeError instead
            of NotImplementedError when the types passed are incorrect. This change means
            there is now consistency with non-overloaded function wrappers which have always
            raised TypeError when the incorrect types are passed. The error message remains
            the same and is for example now:

              TypeError: Wrong number or type of arguments for overloaded function 'f'.
                Possible C/C++ prototypes are:
                  f(int)
                  f(char const *)

            instead of:

              NotImplementedError: Wrong number or type of arguments for overloaded function 'f'.
                Possible C/C++ prototypes are:
                  f(int)
                  f(char const *)

            *** POTENTIAL INCOMPATIBILITY ***"